doc/grpc-proxy-cn.md

```diff
@@ -0,0 +1,53 @@
+[English](grpc-proxy.md)
+# grpc-proxy
+
+通过APISIX代理gRPC连接，并使用APISIX的大部分特性管理你的grpc服务。 
+
+
+
+### 参数
+
+* `service_protocol": "`:  这个 route 的属性`service_protocal` 必须设置为 `grpc`
+* `uri`:   格式为 /service/method 如：/helloworld.Greeter/SayHello
+
+
+
+### 示例
+
+#### 创建代理grpc的route 
+
+在指定 route 中，代理 gRPC 服务接口:
+
+* 注意： 这个 route 的属性`service_protocal` 必须设置为 `grpc`
+* 注意： APISIX 使用TLS加密的HTTP/2暴露gRPC服务, 所以需要先[配置SSL证书](https://github.com/iresty/apisix/blob/master/doc/https-cn.md)
+* 例子所代理的 gRPC 服务可参考：[grpc_server_example](https://github.com/nic-chen/grpc_server_example)
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
+{
+    "methods": ["POST", "GET"],
```

Comment:
for a mini test case, we can remove `"methods": ["POST", "GET"],` this line

Reply:
> for a mini test case, we can remove `"methods": ["POST", "GET"],` this line

can't remove..